Adi.Core.Api.Tests.Unit/Usings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
1. Remove Global Usings
2. Add copyright Header
// ---------------------------------------------------------------------------------- 
// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
// ----------------------------------------------------------------------------------